torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -16,6 +16,19 @@ py::object getFunction(const py::object& module, const char* name) {
   return fn;
 }
 
+// if cachedFn is not none, assign it to local function; otherwise, import
+// function from the scratch and assign it to local function;
+py::object getLocalFunction(const py::object& cachedFn, const char* name) {
+  py::object localFn = py::none();
+  if (cachedFn.is_none()) {
+    localFn =
+        getFunction(py::module::import("torch.distributed.rpc.internal"), name);
+  } else {
+    localFn = cachedFn;
```

Comment:
(Does not need to be fixed in this PR) This will always make a copy of the py::object even if the cachedFn is available. Do you know how expensive it is to make a copy of py::object? It is just similar as creating a shared_ptr? Will it work if we return a const reference here to extend the lifetime of a temporary var?

Reply:
if I understand correctly, py::object is a pointer, so should be cheap. I would rather not to return a const reference to extend lifetime, because I do not know when the const will be destroyed, may hit the same issue as now